cvat-ui/webpack.config.js

```diff
@@ -209,6 +209,14 @@ module.exports = (env) => {
                         from: '../node_modules/onnxruntime-web/dist/*.wasm',
                         to  : 'assets/[name][ext]',
                     },
+                    {
+                        from: 'src/assets/opencv*.js',
+                        to  : 'assets/opencv.js',
```

Comment:
What do you think about creating separate directory for opencv in assets? It would be a mess if will keep adding different files without any managment